frontend/src/fixtures/commonsFixtures.js

```diff
@@ -4,25 +4,31 @@ const commonsFixtures = {
             "id": 5,
             "name": "Seths Common",
             "day": 5,
-            "endDate": "6/11/2021",
+            "startDate": "2022-03-05T15:50:10",
+            "endDate": "2023-03-05T15:50:10",
             "totalPlayers": 50,
             "cowPrice": 15,
+            "milkPrice": 10,
         },
         {
             "id": 4,
             "name": "Kevin's Commons",
             "day": 5,
```

Comment:
I'm okay with leaving the excess properties from the staff, just in case. ISO format looks good this time.